docs/orphaned_and_conflicting.md

```diff
@@ -34,8 +34,13 @@ Depending on your team and on the directory you are processing, you may either g
 
 For an orphaned page, the generic approach consists of the following:
 
-1. Check if there is an active redirect for the corresponding page in English (see https://github.com/mdn/content/blob/main/files/en-us/_redirects.txt)
-2. Depending on the presence of a redirect, use either
+1. Identify the `mdn/content` commit for the deletion using:
+   ```bash
+   git log -- files/en-us/slug/to/document.md
+   ```
+   Then check the corresponding PR to have a better understanding of the change.
```

Comment:
Hi 👋🏻 taking a quick look at the changes.

Could we give hints for the reader for finding the corresponding PR? Should it be:

```
git log -- files/en-us/slug/of/deleted/document.md
```

Something like: find the commit where the document was deleted, search GitHub for the commit and see what the PR summary / context is?

Reply:
Thanks a lot for the feedback @bsmth, I've improved this part (also limiting to the last commit).